src/to-sb2/fake-zip.js

```diff
@@ -0,0 +1,78 @@
+import {assert} from '../util/assert';
+
+import {PNGFile} from '../coders/png-file';
+import {WAVFile} from '../coders/wav-file';
+
+class FakeZipFile {
+    constructor (file) {
+        this.file = file;
+    }
+
+    async (outputType) {
+        assert(outputType === 'uint8array', 'SB1FakeZipFile only supports uint8array');
+
+        return Promise.resolve(this.file.bytes);
+    }
+}
+
+export {FakeZipFile};
+
+class FakeZip {
+    constructor (files) {
+        this.files = files;
+    }
+
+    file (file) {
+        if (file in this.files) {
+            return new FakeZipFile(this.files[file]);
+        }
+    }
+}
+
+export {FakeZip};
+
+const toSb2ImageExtension = imageMedia => {
+    if (imageMedia.extension === 'uncompressed') {
+        return 'png';
+    }
+    return 'jpg';
```

Comment:
Are there actual instances where converting sb1 to sb2 will yield `.jpg`s in the "zip"?

Reply:
There are some instances in the vm where we default to `.png` so I'm wondering if this is going to be a problem...

Reply:
Yes. If you download an sb2 of https://scratch.mit.edu/projects/185346/#editor it will contain a jpg. Loading that sb2 and contained jpg in Scratch 3 seems to work as expected.

Reply:
Ah okay...